src/chrome/content/rules/word.camera.xml

```diff
@@ -0,0 +1,12 @@
+<ruleset name="word.camera">
+	<target host=    "word.camera" />
+	<target host="www.word.camera" />
+
+	<rule from="^http://www\.word\.camera/"
+		to="https://word.camera/" />
+
```

Comment:
It would be nice if you added a comment stating that the cert doesn't match www, for the sake of webmasters reading the Atlas.